smithy-model/src/test/resources/software/amazon/smithy/model/shapes/ast-serialization/cases/box-member-1.0-to.json

```diff
@@ -0,0 +1,157 @@
+{
```

Comment:
are these file names correct?

Reply:
Will rename once we find a better way to load in expected file names for 1.0 roundtrip serialization.